tools/apiview/parsers/cpp-api-parser/ci.yml

```diff
@@ -118,12 +119,30 @@ stages:
                   - checkout: none
                   - download: current
                     artifact: release-artifacts
+                  - pwsh: |
+                      $latestVersion = (azcopy list "https://azuresdkartifacts.blob.core.windows.net/azure-sdk-for-cpp/parseazuresdkcpp/") -replace '[\/;].*' | Get-Unique | sort {[version] $_} -Descending | Select-Object -First 1
```

Comment:
If you do need to parse and sort versions use our semver library for that https://github.com/Azure/azure-sdk-tools/blob/main/eng/common/scripts/SemVer.ps1

Reply:
it should be fine to just use the built-in version type cast though since we don't have any other beta, GA, alpha packages? the format will only be major.minor.build

Reply:
Ok if there is no version in the code then lets just use the build number as the version. We should just use OfficialBuildId https://github.com/Azure/azure-sdk-tools/blob/main/eng/pipelines/templates/variables/globals.yml#L2

Reply:
is the officialBuildId strictly increasing? seems like it's going to be a bit less organized than numerical versioning?

Reply:
It is based on the date and revision of the build pipeline. So yes it will be unique and ever increasing. It is what we base our nuget package alpha versions on.